packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts

```diff
@@ -95,6 +95,146 @@ const singleBarHBCData = [
   },
 ];
 
+const singleBarNMVariantData = [
+  {
+    chartTitle: 'one',
+    chartData: [
+      {
+        legend: 'one',
+        data: 1543,
+        y: 15000,
+        color: '#637cef',
+      },
+      {
+        legend: '',
+        data: 13457,
+        y: 15000,
+        color: '#edebe9',
+        placeholder: true,
+      },
```

Comment:
this is a derived value. Should not be asked from the user.
{
        legend: '',
        data: 13457,
        y: 15000,
        color: '#edebe9',
        placeholder: true,
      },

Reply:
Ohh okay